### DISCUSSION TITLE
Oasis deconvolution when neurons are not active?

### DISCUSSION THREAD

**Discussion by bioBaddie** [flatironinstitute/CaImAn#1323] [Q&A] [answered]
Hi everyone!
I had a question regarding using the oasis deconvolution algorithm on recordings where the majority of neurons are not active. I am finding that when the cells behave nicely (i.e. they have consistent spontaneous activity), the inferred spikes seem accurate, however when the cells are not active I am finding that the deconvolution severely overestimates spikes. Is this expected or did I do something wrong? Do you have any suggestions for these kinds of situations? I have provided a couple plots so you can see what I mean. (The traces shown are from the same neuron, just at different time points during an experiment). I also made the data binary by setting all values greater than 0 equal to 1. 

![example_good_deconv](https://github.com/flatironinstitute/CaImAn/assets/96083173/83566ac9-8d3d-4d17-868f-54dc77d41d33)
![example_bad_deconv](https://github.com/flatironinstitute/CaImAn/assets/96083173/5be952a3-543c-4249-baa6-1e4410a7e4f1)

**Comment by kushalkolar** [comment]
Is that neuron in the bottom panels showing a baseline shift or do you have z-axis drift? It looks like the detrending is breaking down.

**Reply by bioBaddie** [reply]
In these experiments we are injuring the neurons. The top plot shows the neuron before injury, but the bottom plot is after injury. The neuron isn't active anymore post injury so that decay signal is just the video getting darker as time progresses. There shouldn't be z-axis drift, but I'm not the one who collected the recordings, so I guess I can't rule that out for certain. Is that the type of denoised signal you would expect if there were z-axis drift?

**Reply by kushalkolar** [reply, marked as answer]
If the neuron is being injured I'd imagine you have a lot of calcium activity that isn't related to APs and deconvolution models will break down.

**Reply by bioBaddie** [reply]
Thank you! I kind of assumed that the deconvolution wouldn't work properly if the neurons aren't firing action potentials, but wasn't for certain if this was the case. Thank you for the clarification :)
